### PR TITLE
SOV-2739: Develop D2 Lend page - Lend modal

### PR DESCRIPTION
**Pull request by Rickk137**
https://sovryn.atlassian.net/browse/SOV-2739

### CODE REVIEW

File: apps/frontend/src/app/5_pages/LendPage/components/LendModal/CurrentStats.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      <div className="flex gap-8">
+        <div className="mt-6 flex flex-col gap-1 text-sm">
+          <div className="text-gray-30">{t(translations.lending.apy)}</div>
+          <AmountRenderer value={apy} suffix="%" />
+        </div>
+        <div className="mt-6 flex flex-col gap-2 text-sm">
+          <div className="text-gray-30">
+            {t(translations.lending.acceptedCollateral)}
+          </div>
+          <Tooltip content={acceptedList} className={styles.assets}>
```
Comment (pietro-maximoff) [inline apps/frontend/src/app/5_pages/LendPage/components/LendModal/CurrentStats.tsx]:
```suggestion
          <Tooltip trigger={TooltipTrigger.click} content={acceptedList} className={styles.assets}>
```

File: apps/frontend/src/app/5_pages/LendPage/hooks/useHandleLending.ts [hunk 2]
```diff
@@ -65,6 +65,24 @@ export const useHandleLending = (
       console.log('native', native);
       console.log('contract', contract);
 
+      console.log({
```
Comment (pietro-maximoff) [inline apps/frontend/src/app/5_pages/LendPage/hooks/useHandleLending.ts]: if you don't need this info pls remove it

File: apps/frontend/src/app/5_pages/LendPage/components/LendModal/LendingForm.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+      depositLocked,
+    [amount, balance, hasDisclaimerBeenChecked, depositLocked],
+  );
+
+  const [lendApy, setLendApy] = useState<Decimal>(state.apy);
+
+  useEffect(() => {
+    asyncCall(
+      `lending/${state.poolTokenContract.address}/nextSupplyInterestRate/${amount}`,
+      () => state.poolTokenContract.nextSupplyInterestRate(amount),
```
Comment (pietro-maximoff) [inline apps/frontend/src/app/5_pages/LendPage/components/LendModal/LendingForm.tsx:76]: after changing the destination branch of this PR to [`SOV-2738: Develop D2 Lend page`] PR you can just use a hook from `apps/frontend/src/app/5_pages/LendPage/hooks/useGetNextSupplyInterestRate.ts`

**Review by pietro-maximoff — CHANGES_REQUESTED** [review]

**Comment by creed-victor** [comment]
changes already at #479

**Pull request closed without merging**